### ISSUE TITLE
add URL

### ISSUE THREAD

**Issue by 89z** [sgodwincs/uriparse-rs#24]
This repo doesn't have the text "URL" in the description, or topics, so this search wont find this repo:

https://github.com/search?q=url+language:rust

also some crates have URL keyword:

https://github.com/servo/rust-url/blob/v2.2.2/url/Cargo.toml#L12

this one does not:

https://github.com/sgodwincs/uriparse-rs/blob/feffe750d6ee5bfccc7062a3f5a362871e81caf0/Cargo.toml#L2-L11

so these will fail too:

- https://crates.io/keywords/url
- https://crates.io/search?q=url

**Comment by sgodwincs** [comment]
It's a bit disingenuous to put "url" as a keyword, since this crate is for URIs not URLs. If you need a URL, you should use the `url` crate. If you need a URI, you should use this crate.

**Comment by 89z** [comment]
Fair enough, but many people probably don't know the difference, and are just going to randomly search for one or the other. So by omitting one of the terms, you're only hurting yourself
